src/NHibernate.Test/NHSpecificTest/NH3567/Mappings.hbm.xml

```diff
@@ -6,23 +6,23 @@
     <id name="Id" type="Int32">
       <generator class="assigned" />
     </id>
-    <property name="Name" length="2000" />
+    <property name="Name" length="1999" />
```

Comment:
With below change to expand parameters, the size defined by OdbcDriver now actually ends in final parameters, and it causes issues for nvarchar(2000) and above: they get converted to NText by ODBC, causing:
```
ERROR [42000] [Microsoft][SQL Server Native Client 11.0][SQL Server]The data types nvarchar and ntext are incompatible in the equal to operator.
```

So here this is just a workaround for this test which was not about long string. But I fear there is a serious trouble with Odbc, and we should consider removing that size specification which was not actually taken into accont until below change.

Reply:
I have searched about this trouble of obc switching to ntext for nvarchar >= 2000. Only to find someone confirming it [here](https://stackoverflow.com/q/8569844/1178314), but no one seem to have a clue about why is it like this.